calabash-cucumber/lib/calabash-cucumber/wait_helpers.rb

```diff
@@ -212,16 +212,19 @@ def wait_for_elements_exist(elements_arr, options={})
       #   The same options as {Calabash::Cucumber::WaitHelpers::DEFAULT_OPTS} apply.
       # @return [nil] when the condition is satisfied
       # @raise [Calabash::Cucumber::WaitHelpers::WaitError] when the timeout is exceeded
-      def wait_for_element_does_not_exists(element_query, options={})
+      def wait_for_element_does_not_exist(element_query, options={})
```

Comment:
@ark-konopacki 

Close!

```
alias_method(new_name, old_name)
```

I would recommend leaving the original implementation as-is (with the typo) and adding the alias.

If you want to keep the change you made (fix the typo), just reverse the args of `:alias_method`

```
alias_method :wait_for_element_does_not_exists, :wait_for_element_does_not_exist
```